src/AbstractCsv.php

```diff
@@ -323,7 +323,7 @@ public function output(string $filename = null): int
 
         echo $this->output_bom;
 
-        return strlen($this->output_bom) + (int) $this->document->fpassthru();
+        return strlen($this->output_bom) + (int)$this->document->fpassthru();
```

Comment:
Why removing the space ??

Reply:
Well, it's an interesting thing :)
I did it according to my habbits. I've checked PSR once again and haven't find the any clarity about such situation.
I think that [here](https://github.com/php-fig-rectified/fig-rectified-standards/issues/13#issuecomment-324653158) is a pretty good explanation why. I agree with it.
Anyway, according to the inner code style of this project my fix is not neccessary.